static/sass/_patterns_testimonial.scss

```diff
@@ -0,0 +1,73 @@
+@mixin vf-p-testimonial {
```

Comment:
Can you not use the Vanilla pull-quote style? https://docs.vanillaframework.io/en/patterns/pull-quote

Reply:
Its font size is way too big for displaying longer testimonials next to each other.

Reply:
Thought so, might be good to propose a modification to the testimonial pattern in Vanilla